docs/content/gs-custom-keycloak-idp.adoc

```diff
@@ -20,7 +20,7 @@ You are logged in to OpenShift as the `customer-admin` user.
 
 . Log into the RHSSO administration console using the URL and credentials you received in your Welcome email.
 
-. Make sure the `openshift` realm is selected om the top left drop down.
+. Select the `{customer-realm-name}` realm from the top left menu of realms.
```

Comment:
@finp Lets revert this change, as they `openshift` realm is still intended for managing user access to the cluster and the various services.

I think we need a new section or guide for the `customer-applications` realm, which is intended for (probably) any other auth use case the customer has with their own applications. e.g. in a mobile app scenario the end user of the app would be setup in this realm.

Reply:
@david-martin  I've updated, I think the info should be as plain as possible, but maybe I could put an item into Release Notes:
Known issue
* Developer users and application users are defined in the same RHSSO instance. This will be resolved in a future release.

WDYT?

Reply:
It sound reasonabel, but I'd run it past Joe and/or Steph to get their take on wording.